x-pack/auditbeat/module/system/host/_meta/data.json

```diff
@@ -1,46 +1,51 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
+    "agent": {
```

Comment:
This is not part of this PR, right? I'm asking because this change shouldn't get into 6.6.

Reply:
This comes from the Metricbeat testing framework (`[data_generator.go](https://github.com/elastic/beats/blob/master/metricbeat/mb/testing/data_generator.go#L151)`). So I think it should be different in 6.6.